src/lib/useMultikeyFileGenerator.ts

```diff
@@ -80,7 +80,7 @@ export default function useMultikeyFileGenerator({
         ? Just(seed)
         : await attemptNetworkSeedDerivation({
             urbitWallet: seedWallet || urbitWallet,
-            wallet,
+            wallet: wallet,
```

Comment:
this can be shortened to just `wallet`, but also, do we need to pass this param in if `seedWallet` will always exist? We might not need `urbitWallet` either

Reply:
at this point, `seedWallet` is only passed in during the activation flow. we can refactor this to make it a required argument; for now, will just address the duplicate naming